src/components/interaction-designer/Block.vue

```diff
@@ -197,6 +197,13 @@ export default {
       return this.block.exits.length
     },
 
+    visibleExits() {
+      return filter(this.block.exits, function(exit) {
+        const isVisible = get(exit.config, 'is_visible') //TODO: change lodash.get with direct prop access when `is_visible` is in IBlockExit interface
```

Comment:
@bzabos , my proposal here is to add `is_visible` prop in IBlockExit to easily reflect the spec
> This block can be configured to have a single exit, or a number of exits with possibilities

Or should we put it outside `config`, and update `IBlock` interface instead ?

What do you think?

Reply:
After a call with Brett, will avoid using the `is_visible` approach to make sure the block.exits is our source of true for what we have in UI.
Miro note: https://miro.com/app/board/o9J_lXulmu4=/?moveToWidget=3074457356308530338&cot=14

Reply:
@bzabos : it's no longer using `is_visible` approach according to our last chat.